PWGHF/HFC/Tasks/taskFlow.cxx

```diff
@@ -100,90 +145,262 @@ struct HfTaskFlow {
 
   HistogramRegistry registry{"registry"};
 
+  // Correlation containers used for data
   OutputObj<CorrelationContainer> sameTPCTPCChCh{"sameTPCTPCChCh"};
   OutputObj<CorrelationContainer> mixedTPCTPCChCh{"mixedTPCTPCChCh"};
   OutputObj<CorrelationContainer> sameTPCTPCHfCh{"sameTPCTPCHfCh"};
   OutputObj<CorrelationContainer> mixedTPCTPCHfCh{"mixedTPCTPCHfCh"};
   OutputObj<CorrelationContainer> sameTPCMFTChCh{"sameTPCMFTChCh"};
   OutputObj<CorrelationContainer> mixedTPCMFTChCh{"mixedTPCMFTChCh"};
+  OutputObj<CorrelationContainer> sameTPCMFTHfCh{"sameTPCMFTHfCh"};
+  OutputObj<CorrelationContainer> mixedTPCMFTHfCh{"mixedTPCMFTHfCh"};
+
+  // Correlation containers used for Monte-Carlo
+  OutputObj<CorrelationContainer> sameTPCTPCChChMC{"sameTPCTPCChChMC"};
+  OutputObj<CorrelationContainer> mixedTPCTPCChChMC{"mixedTPCTPCChChMC"};
 
   //  =========================
   //      init()
   //  =========================
   void init(InitContext&)
   {
-    //  EVENT HISTOGRAMS
+    //  =========================
+    //      Event histograms
+    // TO-DO : do i have to separate event histograms between DATA and MC ?
+    //  =========================
     constexpr int kNBinsEvents = 3;
     registry.add("Data/hEventCounter", "hEventCounter", {HistType::kTH1F, {{kNBinsEvents, 0.5, 0.5 + kNBinsEvents}}});
     //  set axes of the event counter histogram
     std::string labels[kNBinsEvents];
     labels[0] = "all";
     labels[1] = "after trigger selection (Run 2)";
     labels[2] = "after Physics selection";
+
+    const int maxMixBin = axisMultiplicity->size() * 14; // 14 bins for z-vertex
```

Comment:
Isn't this just the number of mixing bins? If yes, I would suggest to call it `nBinsMix`.

Reply:
I think so too, I renamed it nBinsMix